src/main/java/com/uid2/admin/job/jobsync/PrivateSiteDataSyncJob.java

```diff
@@ -3,6 +3,7 @@
 import com.fasterxml.jackson.databind.ObjectWriter;
```

Comment:
as mentioned in my other comment, we won't enable CSTG for private operator so we don't need to generate PrivateSiteData here for private operator for now.

Reply:
Ok I disabled this with a comment in PrivateSiteDataSyncJob.java
//        keypairSyncJob.execute(); Keypair sync Job should not be executed until we want to enable CSTG for private operators
if that works for you.

Reply:
looks good, do remember we only allow cstg api call for public (not private) operator and only public operator should load keystore data (if and when you will work on operator to load keypair data)